my-app/src/App.js

```diff
@@ -1,14 +1,12 @@
 import React, { Component } from 'react';
+
 import './App.css';
+// importing Board component
+import Board from './Board.js'
+
 //import '../public/vendor/bootstrap/css/bootstrap.min.css'
 import { Switch, Route, Link } from 'react-router-dom'
 
-const Board = () => (
-	<div>
-		<h2>Boards</h2>
-	</div>
-)
-
 const Dashboard = () => (
 	<div class="container">
```

Comment:
Make sure that your spacing is correct. 2 spaces not 4 spaces. Use spaces instead of tabs. you can configure it via your vimrc file or whatever text editor you use